src/gui/folderstatusmodel.cpp

```diff
@@ -934,8 +934,12 @@ void FolderStatusModel::slotSetProgress(const ProgressInfo &progress)
     if (totalSize > 0) {
         QString s1 = Utility::octetsToString( completedSize );
         QString s2 = Utility::octetsToString( totalSize );
+
         //: Example text: "5 minutes left, 12 MB of 345 MB, file 6 of 7"
-        overallSyncString = tr("%5 left, %1 of %2, file %3 of %4")
+        QString msg = progress.trustEta()
+                ? tr("%5 left, %1 of %2, file %3 of %4")
+                : tr("%1 of %2, file %3 of %4");
+        overallSyncString = msg
             .arg(s1, s2)
             .arg(currentFile).arg(totalFileCount)
             .arg( Utility::durationToDescriptiveString1(progress.totalProgress().estimatedEta) );
```

Comment:
I think arg() will show a warning if there is not a corresponding '%',  doesn't it?